javascriptv3/example_code/bedrock-runtime/package.json

```diff
@@ -7,11 +7,10 @@
   "scripts": {
     "integration-test": "vitest run **/*.integration.test.js"
   },
-  "dependencies": {
-    "@aws-sdk/client-bedrock-runtime": "^3.533.0"
-  },
   "devDependencies": {
-    "vitest": "^1.6.0",
-    "zod": "^3.22.4"
+    "vitest": "^1.6.0"
+  },
+  "dependencies": {
+    "@aws-sdk/client-bedrock-runtime": "^3.587.0"
   }
```

Comment:
"type": "module" 
Should be in the top-level package.json.

Reply:
But it is, on line 6.

Reply:
I overlooked it, my mistake.